### PR TITLE
elasticsearch indexing of result

### PR DESCRIPTION
**Pull request by abimbolaomotoso**

### CODE REVIEW

File: src/Microsoft.Crank.Controller/Program.cs [hunk 1]
```diff
@@ -1102,6 +1134,15 @@ string EscapeCsvValue(string value)
                     }
                 }
 
+ if (!String.IsNullOrEmpty(_elasticSearchUrl))
```
Comment (sebastienros) [inline src/Microsoft.Crank.Controller/Program.cs:1137]: Formatting is wrong across the document

File: src/Microsoft.Crank.Controller/Program.cs [hunk 2]
```diff
@@ -36,7 +36,8 @@ public class Program
 
         private static string _tableName = "Benchmarks";
         private static string _sqlConnectionString = "";
-
+ private static string _indexName = "Benchmarks";
```
Comment (sebastienros) [inline src/Microsoft.Crank.Controller/Program.cs:39]: If it's case sensitive then maybe use lowercase?

File: src/Microsoft.Crank.Controller/Program.cs (hunk 1 above)
Comment (abimbolaomotoso) [reply src/Microsoft.Crank.Controller/Program.cs:1137]: oh! I'll attend to it and get back

Comment (abimbolaomotoso) [reply src/Microsoft.Crank.Controller/Program.cs]: Fixed

File: src/Microsoft.Crank.Controller/Program.cs (hunk 2 above)
Comment (abimbolaomotoso) [reply src/Microsoft.Crank.Controller/Program.cs]: Fixed

**Comment by abimbolaomotoso** [comment]
Hello @sebastienros,
I'm I missing anything? Kindly let me know so I can take care of it.
Thanks

**Review by sebastienros — APPROVED** [review]

**Pull request merged**